src/components/filterModal/statsList/BaseEditorLogarithmic.vue

```diff
@@ -3,7 +3,7 @@
         <div class="float-editor_inputs">
             <b-form-input
               type="number"
-              v-model.number="selectedMin"
+              v-model.number="Math.round(selectedMin * 10000) / 10000"
```

Comment:
Check code by eslint. You can add new rules if you want, but it's mandatory to check code by it.